pkg/clusteragent/admission/mutate/tagsfromlabels/tags_test.go

```diff
@@ -270,11 +272,13 @@ func TestGetAndCacheOwner(t *testing.T) {
 	ownerInfo := dummyInfo()
 	kubeObj := newUnstructuredWithSpec(map[string]interface{}{"foo": "bar"})
 	owner := newOwner(kubeObj)
+	wmeta := fxutil.Test[workloadmeta.Component](t, core.MockBundle(), workloadmeta.MockModule(), fx.Supply(workloadmeta.NewParams()))
+	webhook := NewWebhook(wmeta)
 
 	// Cache hit
-	cache.Cache.Set(ownerInfo.buildID(testNamespace), owner, ownerCacheTTL)
+	cache.Cache.Set(ownerInfo.buildID(testNamespace), owner, 10*time.Minute)
```

Comment:
Shouldn't this use `w.ownerCacheTTL()` instead of having `10*time.Minute`?

Reply:
Changed 👍